src/core/components/ErrorPage/index.js

```diff
@@ -2,21 +2,23 @@ import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
-import { getErrorComponent } from 'amo/utils';
+import { getErrorComponent as getErrorComponentCore } from 'core/utils';
```

Comment:
This looks a bit more straightforward to me but I guess either one is slightly awkward:
````js
import { getErrorComponent as defaultErrorComponentGetter } from 'core/utils';
````
Meh, words.